recipe/meta.yaml

```
@@ -26,30 +25,34 @@ requirements:
     - setuptools
   run:
     - python
+    - dnspython >=1.16.0,<3.0.0
 
 test:
   source_files:
     - .
```

```suggestion
    - test
    - pytest.ini
```
We can do without bringing in the whole source code.

---

For some reason, I didn't realize you could pull in individual files.